sqlite/ext/comdb2/functions.c

```diff
@@ -0,0 +1,39 @@
+#include <stdlib.h>
```

Comment:
License missing.

Reply:
adding. also to all the other files in the directory : )